### PR TITLE
Publicized MinecraftClient#textureManager

### PR DESCRIPTION
**Pull request by YTG1234**
Just saw this on #125 so I decided to do it.

### CODE REVIEW

**Comment by YTG1234** [comment]
Maybe we could also move all Forge's access modifications that are already in Patchwork to this new module, idk how it's going to be possible to "load" the access widener when developing other modules. Any thoughts?

File: patchwork-access-modifications/build.gradle [hunk 1]
```diff
@@ -0,0 +1,10 @@
+archivesBaseName = "patchwork-access-modifications"
+version = getSubprojectVersion(project, "0.0.0")
```
Comment (famous1622) [inline patchwork-access-modifications/build.gradle:2]: The version should start at 0.1.0

File: settings.gradle [hunk 2]
```diff
@@ -45,3 +45,4 @@ include 'patchwork-resource'
 include 'patchwork-tags'
 include 'patchwork-tooltype'
 include 'patchwork-vanilla-patches'
+include 'patchwork-access-modifications'
```
Comment (famous1622) [inline settings.gradle:48]: These should be in alphabetical order.

**Review by famous1622 — CHANGES_REQUESTED** [review]

File: settings.gradle (hunk 2 above)
Comment (YTG1234) [reply settings.gradle]: Should be fixed now

File: patchwork-access-modifications/build.gradle (hunk 1 above)
Comment (YTG1234) [reply patchwork-access-modifications/build.gradle]: Should be fixed now

**Comment by TheGlitch76** [comment]
Moving access wideners to their own module isn't the right solution yet because nobody has made a jar processor to make access wideners transitive.
I think instead of its own module, for now this should either be in an appropriate module that already exists (Check the Forge source code and see if it's used anywhere), or failing that dropped in `patchwork-fml`.

**Comment by YTG1234** [comment]
I think it may be used in the forge `CloudRenderer`: `mc.textureManager.bindTexture(texture);`. I'm not sure

**Comment by YTG1234** [comment]
Yep. It's only used in `CloudRenderer`

**Comment by YTG1234** [comment]
I will put it in the FML module

**Comment by YTG1234** [comment]
Moved it to the FML module for now. I didn't bump the version - as the commit msg says.

**Comment by YTG1234** [comment]
Turns out I put a `/`. Fixed it to a `.`, it should run now.

**Comment by YTG1234** [comment]
Updated the title so it's less confusing

**Comment by YTG1234** [comment]
@famous1622 I changed things up a little, can you review again?

**Comment by rikka0w0** [comment]
> Moving access wideners to their own module isn't the right solution yet because nobody has made a jar processor to make access wideners transitive.
> I think instead of its own module, for now this should either be in an appropriate module that already exists (Check the Forge source code and see if it's used anywhere), or failing that dropped in `patchwork-fml`.

I think in the future the rendering stuff should have their own package. But for now I would agree that this aw should be in `patchwork-fml`. Sometimes `mc.textureManager` is used by mods in their tileentity render and entity render, I don't there is a direct reference to that field in Forge.

**Comment by YTG1234** [comment]
It's interesting that in most places Forge uses `getTextureManager()`, so I don't understand why the field has been made public

**Comment by famous1622** [comment]
Was probably public forever ago and they don't want to break mods

**Comment by YTG1234** [comment]
Test

**Review by TheGlitch76 — APPROVED** [review]
LGTM!

**Comment by YTG1234** [comment]
> Was probably public forever ago and they don't want to break mods

I thought Forge was the opposite of backwards-compatibility

**Comment by TheGlitch76** [comment]
Forge is more than just Lex and all of them do try to maintain back-compat for some things.

**Pull request merged**